authentication/pipeline/user.py

```diff
@@ -87,17 +88,54 @@ def create_user_via_email(
     data["email"] = kwargs.get("email", kwargs.get("details", {}).get("email"))
 
     if "name" not in data or "password" not in data:
-        raise RequirePasswordAndProfileException(backend, current_partial)
+        raise RequirePasswordAndAddressException(backend, current_partial)
 
     serializer = UserSerializer(data=data)
 
     if not serializer.is_valid():
-        raise RequirePasswordAndProfileException(
+        raise RequirePasswordAndAddressException(
             backend, current_partial, errors=serializer.errors
         )
     return {"is_new": True, "user": serializer.save()}
 
 
+@partial
+def create_profile_via_email(
```

Comment:
I might be wrong here, but this name seems misleading. `create_user_via_email` has that name because that's the pipeline step that handles user creation from the verification email link, right? If I'm not mistaken, this step handles profile creation from the second page of the registration form, which you don't reach directly from the verification email

Reply:
This step creates a `Profile` mode and follows the `create_user_via_email` pipeline step, so `create_profile_via_email ` seems the most intuitive name to me.

Reply:
Renamed to just `create_profile`